src/ir/function.rs

```diff
@@ -310,12 +313,14 @@ impl FunctionSig {
         return_type: TypeId,
         arguments: Vec<(Option<String>, TypeId)>,
         is_variadic: bool,
+        must_use: bool,
         abi: Abi,
     ) -> Self {
         FunctionSig {
             return_type: return_type,
             argument_types: arguments,
             is_variadic: is_variadic,
+            must_use: must_use,
```

Comment:
I know it's pre-existing, but if you want this can just be `must_use,` (same for the other members).